ui/revenuecatui/src/test/java/com/revenuecat/purchases/ui/revenuecatui/data/processed/TemplateConfigurationFactoryTest.kt

```diff
@@ -0,0 +1,89 @@
+package com.revenuecat.purchases.ui.revenuecatui.data.processed
+
+import android.net.Uri
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.PaywallViewMode
+import com.revenuecat.purchases.ui.revenuecatui.data.TestData
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.Locale
+
+@RunWith(AndroidJUnit4::class)
+internal class TemplateConfigurationFactoryTest {
+
+    private val paywallViewMode = PaywallViewMode.FULL_SCREEN
+
+    private val template2Configuration = TemplateConfigurationFactory.create(
```

Comment:
I don't think Kotlin creates this lazily, does it? I ask because stuff like this is what makes running tests slower, since it might evaluate this even when running a different set of tests.
Maybe it _is_ lazy like Swift, but if not I'd put this in `setUp` instead.

Reply:
It's not lazy no. Will move it there 👍